spec/const3.dd

```diff
@@ -4,209 +4,44 @@ $(D_S Type Qualifiers,
 
 $(HEADERNAV_TOC)
 
-        $(P Type qualifiers modify a type by applying a $(GLINK2 declaration, TypeCtor).
-        $(I TypeCtor)s are: $(D const), $(D immutable), $(D shared), and $(D inout).
-        Each applies transitively to all subtypes.
+        $(P Type qualifiers modify a type by applying a $(I type constructor).
+        $(GLINK2 declaration, TypeCtor)s are:
+        $(D const), $(D immutable), $(D shared), and $(D inout).
```

Comment:
There are many type constructors in D (see definition at https://en.wikipedia.org/wiki/Type_constructor): the array syntax `T[]`, the pointer syntax `T*`, template instantiation `A!T` etc. are all type constructors. 

These are qualifiers (and indeed they are also type constructors), and the types resulting from the application are called qualified types.

Reply:
you're right, I'll fix